content/en/docs/components/model-registry/installation.md

```diff
@@ -16,28 +16,33 @@ These are the minimal requirements to install Model Registry:
 
 <a id="model-registry-install"></a>
 
-## Installing Model Registry
+## Installing Model Registry (as part of Kubeflow)
```

Comment:
@tarilabs Perhaps we could do two H3 subsections, so this becomes

```
## Installing Model Registry

Kubeflow Model registry may be installed as part of a Kubeflow Platform, or as a standalone component.
The best option for you will depend on your specific requirements.

### Kubeflow Platform

Kubeflow Model Registry is available in Kubeflow Platform 1.9+, see [Installing Kubeflow](/docs/started/installing-kubeflow/) to learn more about deploying the Kubeflow Platform.

If you have deployed the raw manifests, you may follow [these instructions]((https://github.com/kubeflow/manifests/tree/master/apps/model-registry/upstream#readme), please raise any feedback on [`kubeflow/manifests#2631`](https://github.com/kubeflow/manifests/issues/2631).

### Standalone

.... Guide ....

Reply:
implemented with https://github.com/kubeflow/website/pull/3740/commits/1542df1079e9cbc13959a1345a8045940b84cd42

thank you; resolving.